.github/workflows/health-check.yaml

```diff
@@ -43,18 +59,24 @@ jobs:
       - name: Free disk space
         uses: ./.github/actions/free-disk-space
 
+      - name: Set Swap Space
+        if: ${{ matrix.platform == 'arm64' }}
+        uses: pierotofy/set-swap-space@0404882bc4666c0ff2f6fd8b3d32af69a730183c
+        with:
+          swap-size-gb: 16
+
       - name: Build 'Autoware'
         uses: ./.github/actions/docker-build
         with:
-          platform: amd64
+          platform: ${{ matrix.arch-platform }}
           cache-tag-suffix: ${{ matrix.build-type }}
           additional-repos: ${{ matrix.build-type == 'nightly' && 'autoware-nightly.repos' || '' }}
           build-args: |
             ROS_DISTRO=${{ needs.load-env.outputs.rosdistro }}
             BASE_IMAGE=${{ needs.load-env.outputs.base_image }}
             AUTOWARE_BASE_IMAGE=${{ needs.load-env.outputs.autoware_base_image }}
             AUTOWARE_BASE_CUDA_IMAGE=${{ needs.load-env.outputs.autoware_base_cuda_image }}
-            LIB_DIR=x86_64
+            LIB_DIR=${{ matrix.lib-dir }}
```

Comment:
https://github.com/autowarefoundation/autoware/actions/runs/14531007806/job/40770632289?pr=6054

docker-build (main) has failed. Only change I can see is the capitalization change of `LIB_DIR=x86_64` in the matrix above.

Reply:
I think it has no capitalization change but I found the GitHub Actions Cache problem and fix it. https://github.com/autowarefoundation/autoware/pull/6054/commits/416b87ecabccbd19077690548ea538489a04e6a3 https://github.com/autowarefoundation/autoware/pull/6054/commits/93edb39903edf215f54dacdf6aa1ad72c3608b0b